src/sentry/backup/findings.py

```diff
@@ -19,21 +18,30 @@ class InstanceID:
     # The order that this model appeared in the JSON inputs. Because we validate that the same
     # number of models of each kind are present on both the left and right side when validating, we
     # can use the ordinal as a unique identifier.
-    ordinal: int | None = None
-
-    def __init__(self, model: NormalizedModelName, ordinal: Optional[int] = None):
-        self.model = str(model)
-        self.ordinal = ordinal
-
-    def __hash__(self):
-        return hash((self.model, self.ordinal))
+    ordinal: Optional[int] = None
 
     def pretty(self) -> str:
         out = f"InstanceID(model: {self.model!r}"
         if self.ordinal:
             out += f", ordinal: {self.ordinal}"
         return out + ")"
 
+    @classmethod
+    def encode(_, id: InstanceID) -> dict:
```

Comment:
I don't see the `encode` and `decode` functions used anywhere. What is it used for?

Reply:
Good catch - these are vestigial from another approach I had taken. They've been removed.